src/you_get/extractors/bilibili.py

```diff
@@ -135,6 +136,15 @@ def test_bili_cdns(urls_list):
     return -1, None, 0
 
 def bilibili_download_by_cid(cid, title, output_dir='.', merge=True, info_only=False):
+        info_u = 'http://interface.bilibili.com/player?id=cid:' + cid
+        video_info = get_content(info_u).split('\n')
+        oriurl = ''
+        for line in video_info:
+            if line.startswith('<oriurl>'):
+                oriurl = line[len('<oriurl>'):-len('</oriurl>')]
+        if len(oriurl) != 0:
+            qq_download(oriurl, output_dir=output_dir, merge=merge, info_only=info_only)
```

Comment:
why would you assume vtype must be qq and call qq_download directly
cant we just dump the oriurl back to the you-get parser